images_test.go

```diff
@@ -93,3 +96,18 @@ func TestShaName(t *testing.T) {
 		}
 	}
 }
+
+func TestImagesOrderedAlphabetically(t *testing.T) {
```

Comment:
You can imagine how much I like that. xD

Reply:
This PR is a love letter to you <3